system-tests/project-fixtures/runner-specs/cypress/e2e/runner/ui-states/commands.cy.js

```diff
@@ -0,0 +1,137 @@
+describe('Commands', () => {
+  it('commands that dont display in UI', () => {
+    cy.visit('cypress/fixtures/commandsActions.html')
+
+    cy.wrap({ foo: { bar: 'baz' } })
+    .then((obj) => obj)
+    .should('have.property', 'foo')
+    .and('have.property', 'bar')
+
+    cy.wrap({ foo: { bar: 'baz' } })
+    .as('myObject')
+
+    cy.get('@myObject').then((obj) => {
+      cy.log(obj)
+    })
+
+    cy.get('div').each(($div) => { }).end()
+
+    cy.fixture('uiStates')
+
+    cy.intercept('GET', 'comments/*').as('getComment')
+
+    cy.wrap(['foo', 'bar']).spread(() => {})
+
+    const obj = {
+      foo () { },
+      bar () { },
+    }
+
+    cy.spy(obj, 'foo')
+    cy.stub(obj, 'bar')
+  })
+
+  it('command options - part 1', () => {
```

Comment:
Is the goal here mostly to just set a subset of commands to get the snapshots? I'm not sure what is too different from `commands.cy.js` and `commandsToDisplay.cy.js`. Is it possible to get more descriptive `describe` and `it` naming as to what exactly we are testing? It currently seems too vague.